jscomp/syntax/ast_exp_apply.ml

```diff
@@ -102,6 +102,7 @@ let app_exp_mapper
         a |. M.(f b c) --> M.f a M.b M.c
         a |. (g |. b)
         a |. M.Some
+        a |. M.`Variant
       *)
```

Comment:
This docs seems to be a bit misleading, poly var is not qualified by module in general (structural typing)

Reply:
Indeed, nice catch! Corrected by using Github edit suggestion